### ISSUE TITLE
Compatibility with Repetier 1.0.2? "Not SeeMeCNC"

### ISSUE THREAD

**Issue by Metallikahn** [geneb/OctoPrint-Delta-Calibration#5]
So, I just did a few upgrades to my V2/V3 Hybrid (Titan extruder and a PT-100 temp sensor) and I figured I'd go for a firmware update while I was at it. I ran through the Repetier Online configuration, while pouring over my current firmware/eeprom settings to make sure I had everything filled in exactly the way it was supposed to be. Even made sure to match up the Manufacturer and Machine definitions in Config.h so that it would be recognized as such... At least I thought I did. When the printer boots it shows: SeeMeCNC Rostock MAX V3 and the firmware version, 1.0.2, but the plugin won't recognize it as a SeeMeCNC machine. Is there somewhere else in the firmware I need to update so it's recognized, or is it just not compatible with this latest firmware?

The new firmware has it's own built in configuration (which looks pretty snazzy), but I haven't had the chance to try to make it work with the HE280 accelerometer board yet (if it even can).

Any thoughts?

**Comment by geneb** [comment]
On Sun, 12 Aug 2018, Metallikahn wrote:

> The new firmware has it's own built in configuration (which looks pretty 
> snazzy), but I haven't had the chance to try to make it work with the 
> HE280 accelerometer board yet (if it even can).

The only firmware that will work with the HE280 is the SeeMeCNC fork of 
Repetier Firmware.  The stock firmware won't work, sorry.

g.

-- 
Proud owner of F-15C 80-0007
http://www.f15sim.com - The only one of its kind.
http://www.diy-cockpits.org/coll - Go Collimated or Go Home.
Some people collect things for a hobby.  Geeks collect hobbies.

ScarletDME - The red hot Data Management Environment
A Multi-Value database for the masses, not the classes.
http://scarlet.deltasoft.com - Get it _today_!

**Comment by Metallikahn** [comment]
Well... that sucks. Especially considering how that firmware branch has seemingly been abandoned by SeeMeCNC. No actual updates in over 8 months and no actual version updates in over 2 years! I need at least version 92.8 to get the PT-100 sensor working, but using anything other than their firmware branch makes the autocalibration accelerometer completely useless.

Any idea how I can add a "EXT0_TEMPSENSOR_TYPE" to the 92.2 firmware that would recognize and correctly use the PT-100? I'm currently trying to find the bit of code in the 1.0.2 firmware and copy it over, but really have no idea where it would be located and exactly what I would need to copy. 

I know it's not really related to the issue I posted, but any help would be greatly appreciated!

**Comment by geneb** [comment]
On Sun, 12 Aug 2018, Metallikahn wrote:

> Well... that sucks. Especially considering how that firmware branch has 
> seemingly been abandoned by SeeMeCNC. No actual updates in over 8 months 
> and no actual version updates in over 2 years! I need at least version 
> 92.8 to get the PT-100 sensor working, but using anything other than 
> their firmware branch makes the autocalibration accelerometer completely 
> useless.
>
The code for the HE280 precludes any other updates.  The binary size is on 
the ragged edge of "too big" and will only properly function if compiled 
by the Arduino 1.6.0 IDE.  Compiler & library changes after that point 
make the firmware unstable and prone to freezing.

Your best bet in the long run is to get a Duet WiFi or Ethernet and get an 
SE300 hot end.  The new hot end board has a strain gauge in it for doing 
probing.  It's based on the Duet Smart Effector strain gauge design.

g.


-- 
Proud owner of F-15C 80-0007
http://www.f15sim.com - The only one of its kind.
http://www.diy-cockpits.org/coll - Go Collimated or Go Home.
Some people collect things for a hobby.  Geeks collect hobbies.

ScarletDME - The red hot Data Management Environment
A Multi-Value database for the masses, not the classes.
http://scarlet.deltasoft.com - Get it _today_!

**Comment by Metallikahn** [comment]
Yeah... Best bet for another $200! lol ;-)  It's definitely on my list at some point, but I just got done building a Prusa MK3 and upgrading the extruder to a titan on the Rostock. Between that and filament for my other projects, I'm going to have to hold off for awhile. Really wishing I'd not bought the HE280 in the first place (all I'm using off it is the accelerometer at the moment). FSR's and the Duet would have been a much better investment for sure.

I did get the PT-100 to work finally. I found where someone on the SeeMeCNC forums (what the heck is up with that forum lately, btw? Somehow my Password got screwed up in my login manager and I've been waiting for a reset email, or 6, for 3 days now. Seems like it's turning into a real ghost town) where somebody replaced the thermistor 1 table with a PT-100 table from a Marlin firmware and then set TYPE to 1. that got me working, but I still think whatever table they were using in 1.0.2 was a more accurate one. I'm a tad spikey (though within +/- .3 degrees... Pretty dang good) yet, whereas it was within .1 and very smooth on the 1.0.2 firmware built in table... whichever one that might be. still trying to figure that out, then I should be able to get it added in and working. :-)

Thanks for the help geneb! I really appreciate it! I'll go ahead and close out this issue, since it wasn't really wasn't an issue, so much as lack of knowledge on my part. lol 

Thanks again!
